pandas/core/window.py

```diff
@@ -944,13 +944,13 @@ def count(self):
         return self._wrap_results(results, blocks, obj)
 
     _shared_docs['apply'] = dedent(r"""
-    %(name)s function apply.
+    The %(name)s function apply.
 
     Parameters
     ----------
     func : function
         Must produce a single value from an ndarray input if ``raw=True``
-        or a Series if ``raw=False``
+        or a Series if ``raw=False``.
     raw : bool, default None
         * ``False`` : passes each row or column as a Series to the
```

Comment:
Since the backticks here cause an error with the validator I think it would be better just to remove unless @datapythonista feels otherwise

Reply:
The problem are not the backticks, it's starting the description with a list. it happens in many cases, and we'll eventually fix the validation docstring, as this is correct.

Same as the errors because having `*args, **kwargs` together. Changes are ok, we'll fix the validation script.